RecoLocalTracker/Phase2TrackerRecHits/src/Phase2StripCPEDummy.cc

```diff
@@ -0,0 +1,17 @@
+
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEDummy.h"
+
+
+Phase2StripCPEDummy::LocalValues Phase2StripCPEDummy::localParameters(const Phase2TrackerCluster1D & cluster, const GeomDetUnit & det) const {
+  float strippitch  = 0.0090; // hardcoded dummy, a la 2S
```

Comment:
Will these be used really only for 2S modules?
PS-s part is 0.01x2.5 .. and PS-p is probably not covered by this CPE (or it can be, but then the length is 0.15)
Please clarify.